### PR TITLE
[5.1] Created validation not exists

### PR DESCRIPTION
**Pull request by moura137**

### CODE REVIEW

File: src/Illuminate/Validation/Validator.php [hunk 1]
```diff
@@ -1097,6 +1097,19 @@ protected function validateExists($attribute, $value, $parameters)
     }
 
     /**
+     * Validate the existence of an attribute value in a database table.
+     *
+     * @param  string  $attribute
+     * @param  mixed   $value
+     * @param  array   $parameters
```
Comment (GrahamCampbell) [inline src/Illuminate/Validation/Validator.php]: don't align please

Comment (moura137) [inline src/Illuminate/Validation/Validator.php]: @GrahamCampbell, I not understand, what should I not align?

Comment (GrahamCampbell) [inline src/Illuminate/Validation/Validator.php]: The phpdoc. You should have exactly 2 spaces either side of the type, and no more.

**Comment by taylorotwell** [comment]
Isn't this just "unique"?

**Comment by M0rbid** [comment]
Taylor, i really dont understand the purpose of 'unique'. This check only make sense when we insert/update this data. so (before: marking column in table as uniaue key) after retrieving insert/update result will know unique it was or not. I belive doing things like:

if ([UPDATE|INSERT] things...) 
{
} else {
sorry.. "thats not unique value for @fieldname" u ll not see but "somthing goes wrong"
}

much better than:

if (SELECT COUNT(*) FROM table WHERE field = $comesFromRequest) 
{
sorry.. thats not unique value for @fieldname
} 
else 
{
[UPDATE|INSERT] things...
}

I think some part of responsibility SHOULD lay on Database.
Other way, you let them use 'unique' rule even on non indexed (and not unique) field, what i think is bad practise.
Any way this logic (touching DB) is "not very about" validation.

Sorry for English

**Pull request closed without merging**